mlrun/config.py

```diff
@@ -544,6 +544,11 @@
         # interval for stopping log collection for runs which are in a terminal state
         "stop_logs_interval": 3600,
     },
+    "events": {
+        "mode": "disabled",
+        "verbose": False,
+        "access_key": "",
+    },
```

Comment:
add some comments to explain behavior / or

```suggestion
    
    # Events are currently (and only) used to audit changes and record access to MLRun entities (such as secrets)
    "events": {
    
        # supported modes "enabled", "disabled".
        # "enabled" - events are emitted.
        # "disabled" - a nop client is used (aka doing nothing).
        "mode": "disabled",
        "verbose": False,
        
        # used for igz client when emitting events
        "access_key": "",
    },
```